BatchManagement.json

```diff
@@ -0,0 +1,1618 @@
+{
+  "swagger": "2.0",
```

Comment:
Oops, this file should not have been included, and has been removed. The diff was only on the BatchManagement.json file inside 2017-05-01. (Only change should be the checkNameAvailability changes).